tests/test_schema.py

```diff
@@ -584,3 +592,540 @@ def test_authentication(self):
 
         msg = executed['errors'][0]['message']
         self.assertEqual(msg, AUTHENTICATION_ERROR)
+
+
+class TestAddEcosystemMutation(django.test.TestCase):
+    """Unit tests for mutation to add ecosystems"""
+
+    BT_ADD_ECO = """
+      mutation addEco {
+        addEcosystem(name: "Example",
+                     title: "Example title",
+                     description: "Example desc.")
+        {
+          ecosystem {
+            id
+            name
+            title
+            description
+          }
+        }
+      }
+    """
```

Comment:
I think it would be better if we only define the query once and then we pass the parameters to it to test it. You have some examples in sortinghat:
https://github.com/chaoss/grimoirelab-sortinghat/blob/muggle/tests/cli/test_cmd_show.py#L33
https://github.com/chaoss/grimoirelab-sortinghat/blob/muggle/tests/cli/test_cmd_show.py#L377

Reply:
Yes, this is how it is defined in the tests for `UpdateEcosysteMutation`. I defined several versions of the query to try what happens when some parameters are ommited. For instance, when the name is ommited it should raise an error, but if the title or the description are, it should set `None` as a default value. Does this make sense?

Reply:
Yes, it does.

Reply:
There were some versions of the query which could be aggregated into the same one, and so I did!